db/021-add-siren-particle-id.sql

```diff
@@ -0,0 +1,20 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 20;
```

Comment:
This ID should match the number in the file name.

Reply:
Good catch!